.github/workflows/build.yml

```diff
@@ -74,7 +78,7 @@ jobs:
           current_sha=$(git rev-parse --short HEAD)
           echo "CURRENT_SHA=$current_sha" >> $GITHUB_ENV
 
-          prev_commit_count=$(echo "${{ steps.last_release.outputs.tag_name }}" | sed -e "s/^r//")
+          prev_commit_count=$(echo "${{ env.TAG_NAME }}" | sed -e "s/^r//")
```

Comment:
Could just put the `gh` command above this line and remove the previous step.

And there's raw REST API call below with `curl` that you could replace with `gh api` so it's cleaner :)

Reply:
I'll leave the api call for the next time